plugins/optimization-detective/site-health.php

```diff
@@ -0,0 +1,171 @@
+<?php
+/**
+ * Site Health checks.
+ *
+ * @package optimization-detective
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Adds the Optimization Detective REST API check to site health tests.
+ *
+ * @since n.e.x.t
+ *
+ * @param array{direct: array<string, array{label: string, test: string}>} $tests Site Health Tests.
+ * @return array{direct: array<string, array{label: string, test: string}>} Amended tests.
+ */
+function od_optimization_detective_add_rest_api_test( array $tests ): array {
+	$tests['direct']['optimization_detective_rest_api'] = array(
+		'label' => __( 'Optimization Detective REST API Endpoint Availability', 'optimization-detective' ),
+		'test'  => 'od_optimization_detective_rest_api_test',
+	);
+
+	return $tests;
+}
+
+/**
+ * Tests availability of the Optimization Detective REST API endpoint.
+ *
+ * @since n.e.x.t
+ *
+ * @return array{label: string, status: string, badge: array{label: string, color: string}, description: string, actions: string, test: string} Result.
+ */
+function od_optimization_detective_rest_api_test(): array {
+	$result = array(
+		'label'       => __( 'The REST API endpoint is functional.', 'optimization-detective' ),
```

Comment:
Should this not say "The Optimization Detective REST API endpoint" instead of "The REST API endpoint". I get there's the "Optimization Detective" badge next to it, but that's still not a super helpful label without including that.